src/NIDAQ.jl

```diff
@@ -41,12 +41,12 @@ export Bool32
 
 try
   global ver
-  major = Uint32[0]
-  ccall((:DAQmxGetSysNIDAQMajorVersion,NIDAQmx),Int32,(Ptr{Uint32},),major)
-  minor = Uint32[0]
-  ccall((:DAQmxGetSysNIDAQMinorVersion,NIDAQmx),Int32,(Ptr{Uint32},),minor)
-  update = Uint32[0]
-  ccall((:DAQmxGetSysNIDAQUpdateVersion,NIDAQmx),Int32,(Ptr{Uint32},),update)
+  major = UInt32[0]
```

Comment:
Ben,
could you please help me with installation of Clang.jl. 
I'm getting following error:
'''julia
julia> Pkg.build("Clang")                                                       
INFO: Building Clang                                                            
which: no llvm-config in (/cygdrive/c/Users/phlavenk/AppData/Local/Julia-0.4.0/b
in:/cygdrive/c/Users/phlavenk/AppData/Local/Julia-0.4.0/Git/bin:/cygdrive/c/Prog
ram Files/ConEmu:/cygdrive/c/Program Files/ConEmu/ConEmu:/cygdrive/c/ProgramData
/Oracle/Java/javapath:/cygdrive/c/Program Files/ImageMagick-6.8.9-Q16:/cygdrive/
c/WINDOWS/system32:/cygdrive/c/WINDOWS:/cygdrive/c/WINDOWS/System32/Wbem:/cygdri
ve/c/WINDOWS/System32/WindowsPowerShell/v1.0:/cygdrive/c/Program Files/Intel/WiF
i/bin:/cygdrive/c/Program Files/Common Files/Intel/WirelessCommon:/cygdrive/c/Pr
ogram Files/Microsoft SQL Server/110/Tools/Binn:/cygdrive/c/Program Files (x86)/
CMake/bin:/cygdrive/c/Program Files (x86)/Windows Kits/8.1/Windows Performance T
oolkit:/cygdrive/c/Program Files (x86)/Microsoft SDKs/TypeScript/1.0:/cygdrive/c
/Program Files/Microsoft SQL Server/120/Tools/Binn:/cygdrive/c/Program Files (x8
6)/Git/cmd:/cygdrive/c/Program Files (x86)/Notepad++:/cygdrive/c/Users/phlavenk/
AppData/Local/Continuum/Anaconda:/cygdrive/c/Users/phlavenk/AppData/Local/Contin
uum/Anaconda/Scripts:/cygdrive/c/Program Files (x86)/Tools:/cygdrive/c/Users/phl
avenk/AppData/Local/Julia-0.4.0/bin:/cygdrive/d/Geant4/xerces-c-3.1.1-x86-window
s-vc-10.0/bin:/usr/bin:/cygdrive/c/Program Files/HDF_Group/HDFView/2.11.0/bin:/c
ygdrive/d/Geant4/QT4/bin:/cygdrive/c/root/bin:/cygdrive/c/Users/phlavenk/AppData
/Local/atom/bin)                                                                
================================[ ERROR: Clang ]================================
                                                                                
                                                                                
LoadError: failed process: Process(`which llvm-config`, ProcessExited(1)) [1]   
while loading c:\Users\phlavenk\JuliaData\v0.4\Clang\deps\build.jl, in expressio
n starting on line 12                                                           
                                                                                
================================================================================
                                                                                
                                                                                
================================[ BUILD ERRORS ]================================
                                                                                
                                                                                
WARNING: Clang had build errors.                                                
                                                                                
 - packages with build errors remain installed in c:\Users\phlavenk\JuliaData\v0
.4                                                                              
 - build the package(s) and all dependencies with `Pkg.build("Clang")`          
 - build a single package by running its `deps/build.jl` script                 
                                                                                
================================================================================
'''

Seems to me LLVM libs are missing and are not downloaded as a Clang.jl dependecy.. Could you please tell me the steps to get LLVM running on windows so it's recognized by Julia Pkg ecosystem?
Thanks a lot.
Petr